build/index.js

```diff
@@ -1,19 +1,30 @@
-import Experiment from '../es6/experiment';
-import Interpreter from '../es6/interpreter';
-import Random from '../es6/ops/random';
-import Core from '../es6/ops/core';
-import * as Namespace from '../es6/namespace';
-import Assignment from '../es6/assignment';
-import ExperimentSetup from '../es6/experimentSetup';
+import provideAssignment from '../es6/assignment';
+import provideExperiment from '../es6/experiment';
+import * as ExperimentSetup from '../es6/experimentSetup';
+import provideInterpreter from '../es6/interpreter';
+import * as Random from '../es6/ops/random';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
+import provideNamespace from '../es6/namespace';
+
+// Provide our operations to the OpsUtils module
+OpsUtils.initializeOperators(Core, Random);
```

Comment:
It'd be great to standardize this boilerplate for creating a planout bundle so that we can enforce that ```index.js``` and ```index_core_compatible.js``` stay in sync.

@jcwilson How do you feel about something like this?

**planoutAPIFactory.js**

```javascript
import provideAssignment from '../es6/assignment';
import provideExperiment from '../es6/experiment';
import * as ExperimentSetup from '../es6/experimentSetup';
import provideInterpreter from '../es6/interpreter';
import * as Base from '../es6/ops/base';
import * as Core from '../es6/ops/core';
import * as OpsUtils from '../es6/ops/utils';
import provideNamespace from '../es6/namespace';

export default ({
  Random = null
} = {}) => {
  // Provide our operations to the OpsUtils module
  OpsUtils.initializeOperators(Core, Random);

  // Inject our Random and other dependencies into our modules
  const Assignment = provideAssignment(Random);
  const Experiment = provideExperiment(Assignment);
  const Interpreter = provideInterpreter(OpsUtils, Assignment);
  const Namespace = provideNamespace(Random, Assignment, Experiment);

  return {
    Assignment,
    Experiment,
    ExperimentSetup,
    Interpreter,
    Ops: {
      Random,
      Core,
      Base
    },
    Namespace
  };
};
```

**index.js**
```javascript
import planoutAPIFactory from './planoutAPIFactory';
import * as Random from '../es6/ops/random';
```

export default planoutAPIFactory({Random});

**index_core_compatible.js**
```javascript
import planoutAPIFactory from './planoutAPIFactory';
import * as Random from '../es6/ops/randomPlanoutCoreCompatible';

export default planoutAPIFactory({Random});
```